### PR TITLE
Code Moved From API Project

### PR DESCRIPTION
**Pull request by ocallm1**
- [ ] I have read the [CONTRIBUTION GUIDE][contributing]

- [ ] This is a bug

- [ ] This is a feature request

- [ ] This is a syntax/style fix

- [X ] This is an improvement to code quality

---

**Please describe your changes below:**
(Note: A clear description of changes is required to approve any Pull Requests)

_"Changed the implementation for verification because..."_

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>root-parent</artifactId>
+        <groupId>com.dell.cpsd</groupId>
+        <version>1.0.1</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.dell.cpsd.service.rcm</groupId>
```
Comment (giddyelysium) [inline pom.xml]: the recommendation here is to name this as com.dell.cpsd (only groupID approved)

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    <version>0.1.0-SNAPSHOT</version>
+
+    <licenses>
+        <license>
+            <name>Eclipse Public License - v 1.0</name>
+            <url>http://www.eclipse.org/legal/epl-v10.html</url>
+        </license>
+    </licenses>
+
+    <developers>
```
Comment (giddyelysium) [inline pom.xml]: Add our name to the developers listing

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+            <organization>com.dell.cpsd</organization>
+            <organizationUrl>https://github.com/rousef42</organizationUrl>
+        </developer>
+    </developers>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.dell.cpsd</groupId>
+                <artifactId>common-dependencies</artifactId>
```
Comment (giddyelysium) [inline pom.xml]: Try and update to use the latest release version from maven central

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+            <groupId>com.dell.cpsd.service.rcm</groupId>
+            <artifactId>rcm-definition-service-api</artifactId>
+            <classifier>schemas</classifier>
+            <type>zip</type>
+            <version>1.0.2-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.dell.cpsd.common.messaging</groupId>
+            <artifactId>common-rabbitmq</artifactId>
```
Comment (giddyelysium) [inline pom.xml]: Try and update to use the latest release version from maven central

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+            <version>1.4-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.dell.cpsd.common.client</groupId>
+            <artifactId>common-client</artifactId>
+            <version>1.3-SNAPSHOT</version>
```
Comment (giddyelysium) [inline pom.xml]: Try and update to use the latest release version from maven central

Comment (ocallm1) [reply pom.xml]: Did all those updates now Des

**Pull request merged**